generate-templates.yaml

```diff
@@ -62,7 +62,7 @@
   - name: Generate RHEL 8 templates
     template:
       src: rhel8.tpl.yaml
-      dest: "{{ playbook_dir }}/dist/templates/{{ os }}-{{ item.workload }}-{{ item.flavor }}.yaml"
+      dest: "{{ playbook_dir }}/dist/templates/{{ os }}-{{ item.workload }}-{{ item.flavor }}{{ '' if target_arch == 'x86_64' else '-' + target_arch }}.yaml"
```

Comment:
Do we want to separate the generated templates also by path?

Reply:
We can do it if you’d like, but this would require changes to the following commits as well. Ultimately, the release will consist of three files: one containing all architectures and one for each supported architecture.

What are your thoughts, @akrejcir, @jcanocan, @ksimon1, @lyarwood?

Reply:
> Ultimately, the release will consist of three files: one containing all architectures and one for each supported architecture.

As long as it integrates nicely with SSP we can keep it that way.